src/components/profile/default/DefaultProfileButtons.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+// MUI
+import Button from '@material-ui/core/Button';
+import withStyles from '@material-ui/core/styles/withStyles';
+import style from '../../../style/style';
+
+const DefaultProfileButtons = props => {
```

Comment:
update all these components to use this simpler extraction pattern:
/**
* //blablabla
* @param {object} props
* @param {any} props.classes
*/
= ({classes}) =>{

Reply:
fixed